### DISCUSSION TITLE
Freezing grafana release image version

### DISCUSSION THREAD

**Discussion by sparvata** [teslamate-org/teslamate#3340] [Q&A] [answered]
Hi,

I would like to freeze the installed grafana version to whatever the current installed version is. I have tried the version number 8.5.26 in the yaml file like below but when I pulled I got errors saying this version was not found. Anyone has any ideas?

image: teslamate/grafana:8.5.26
Also tried
image: teslamate/grafana:v8.5.26

only this one works
image: teslamate/grafana:latest

**Comment by cwanja** [comment, marked as answer]
Have you tried to mirror the TeslaMate version? Unless you were running grafana/grafana, you should match the TeslaMate version. 

https://hub.docker.com/r/teslamate/grafana/tags

Thus it would be `teslamate/grafana:1.27.3`

**Reply by sparvata** [reply]
Thank you very much. That worked without issues.
